hal/Cargo.toml

```diff
@@ -20,17 +20,25 @@ features = ["samd21g", "samd21g-rt", "unproven", "usb"]
 
 [dependencies]
 bitfield = "0.13"
+bitflags = "1.2.1"
 cortex-m = "0.6"
 embedded-hal = "0.2"
+generic-array = "0.14.4"
 nb = "0.1"
 paste = "1.0"
 rand_core = "0.5"
+seq-macro = "0.2.1"
+typenum = "1.12"
```

Comment:
`generic-array` re-exports `typenum`. Maybe it would be better to use that to avoid accidental version mismatches in the future? Not a major problem in my opinion, just a thought.

Reply:
Yeah, that's a good idea. I don't really need a specific version of either. I'll do that.